recipe/meta.yaml

```diff
@@ -63,7 +64,8 @@ requirements:
     - libpng >=1.6.23,<1.7
     - openblas 0.2.19|0.2.19.*     # [not win]
     - blas 1.1 {{ blas_variant }}  # [not win]
-    - ffmpeg                       # [not win]
+    - ffmpeg >=3.2.3,<3.2.6        # [not win]
```

Comment:
And unfortunately ffmpeg tends to add new symbols even on patch versions. So there's no strong guarantee that a future 3.2.6 would have the same symbols as 3.2.3, 3.2.4 and 3.2.5